sbin/init/rc.d/local_openntpd

```diff
@@ -45,4 +45,10 @@ local_openntpd_checkconfig()
 	eval ${command} ${command_args} ${local_openntpd_flags} -n
 }
 
+local_openntpd_precmd()
+{
+	local_openntpd_checkconfig
+	chown 0:0 /var/db/ntp
+}
```

Comment:
Seems like this  is not the  right place to do this...

It would be better to handle this from mtree or at install time.
I'm strongly against chowning random file to root at boot time...

Reply:
I would like to do it properly, but I just don't see any way to do it.

/var/db/ntp is chown'd in https://github.com/HardenedBSD/hardenedBSD/blob/hardened/current/master/etc/mtree/BSD.var.dist. I don't know of any way to do some conditional instruction depending on src.conf options.

An alternative would be running chown during installworld. Are you ok with that?